### ISSUE TITLE
Analysen nach Scholienbüchlein?

### ISSUE THREAD

**Issue by MerlinB** [valuehack/scholarium.at#110]

**Comment by rahim-t** [comment]
Nein, später werden Scholienbüchlein und Analysen als eigene Schriften in der e-Bibliothek zusammengeführt. Dzt. sollten die Analysen als Bücher geführt werden, die bestellbar sind.

**Comment by rahim-t** [comment]
Einstweilen Scholien01/2017 zu ergänzen (20€)
